react/features/old-client-notification/functions.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+/**
+ * Returns true if Jitsi Meet is running in too old jitsi-meet-electron app and false otherwise.
+ *
+ * @returns {boolean} - True if Jitsi Meet is running in too old jitsi-meet-electron app and false otherwise.
+ */
+export function isOldJitsiMeetElectronApp() {
+    const match = navigator.userAgent.match(/(JitsiMeet)\s*\/\s*((\d+)\.[^\s]*)/);
```

Comment:
Isn't there a specific Electron global we could use to confirm it's Electron? Just to be double-safe.

Reply:
used the browser check that we have